platform-hub-api/app/services/feature_flag_service.rb

```diff
@@ -0,0 +1,43 @@
+module FeatureFlagService
+  extend self
+
+  # Feature flags data is stored in HashRecord as:
+  # data: {
+  #   'your-feature-flag-name': Bool,
```

Comment:
Maybe this example should use underscores, to better represent how we'll actually be using this?